browser/js/calculator.js

```diff
@@ -0,0 +1,250 @@
+( function() {
```

Comment:
@deadlyicon I was just wondering about whether or not a iify is the best choice here. This looks all good, it's more of a general question about why an iify would be a good/bad choice.